### PR TITLE
Fix/add pip config ios

### PR DESCRIPTION
**Pull request by g7i**
#### Describe the changes
Add audio configs required for PIP for iOS

### CODE REVIEW

**Comment by g7i** [comment]
fixes #2561

**Comment by nickfujita** [comment]
@g7i would you mind testing this PR for the iOS conversion to swift? It does not have the changes you have made here, but I'd be happy to add them in afterwards. I'm looking to have anyone who has contributed to ios test out the converted version. Please be aware that there is a required update to add `use_frameworks! :linkage => :static` to your project Podfile
https://github.com/react-native-video/react-native-video/pull/2527

**Comment by keyberth-rengel** [comment]
how's it going? Hey, how could you do picture-in-picture? I've been trying for many months but I haven't found any information. Could you help me?

File: ios/Video/RCTVideo.m [hunk 1]
```diff
@@ -908,8 +908,11 @@ - (void)setRestoreUserInterfaceForPIPStopCompletionHandler:(BOOL)restore
 
 - (void)setupPipController {
   if (!_pipController && _playerLayer && [AVPictureInPictureController isPictureInPictureSupported]) {
+    [[AVAudioSession sharedInstance] setActive:YES error:nil];
+    [[AVAudioSession sharedInstance] setCategory:AVAudioSessionCategoryPlayback error:nil];
     // Create new controller passing reference to the AVPlayerLayer
     _pipController = [[AVPictureInPictureController alloc] initWithPlayerLayer:_playerLayer];
+    _pipController.canStartPictureInPictureAutomaticallyFromInline = true;
```
Comment (mattleff) [inline ios/Video/RCTVideo.m:915]:
```suggestion
    if (@available(iOS 14.2, *)) {
        _pipController.canStartPictureInPictureAutomaticallyFromInline = true;
    }
```

**Review by mattleff — CHANGES_REQUESTED** [review]
This change should be guarded by an iOS version check

**Comment by hueniverse** [comment]
@g7i Thank you for the PR. Can you review @mattleff suggestion? Also, how can we test this change?

**Comment by freeboub** [comment]
> @g7i Thank you for the PR. Can you review @mattleff suggestion? Also, how can we test this change?

It can be enabled by default in basic sample ;)

**Comment by freeboub** [comment]
> @g7i Thank you for the PR. Can you review @mattleff suggestion? Also, how can we test this change?

It can be enabled by default in basic sample ;)

**Comment by hueniverse** [comment]
Closing due to lack of response (from the original submitted or anyone else to review and rebase). If someone wants to pick this up please open a new PR that ports this code into the current master. We are doing our best to start clean with v6 which requires being a bit aggressive in closing stale PRs that are over a year old. Please don't consider this dismissing of your work and contribution - it's greatly appreciated.

**Pull request closed without merging**